### PR TITLE
Radiometric Target IHM

### PR DESCRIPTION
**Pull request by vincentmigot**
Add interfaces for radiometric targets:
- index: display list of radiometric targets
- create: display form to create a radiometric target
- view: display the detail of a radiometric target

### CODE REVIEW

File: models/yiiModels/RadiometricTargetSearch.php [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+// SILEX-PHIS
+// Copyright © INRA 2018
+// Creation date: 27 September, 2018
+// Contact: vincent.migot@inra.fr, anne.tireau@inra.fr, pascal.neveu@inra.fr
+//******************************************************************************
+
+namespace app\models\yiiModels;
+
+/**
+ * implements the search action for the radioemtric target
```
Comment (annetireau) [inline models/yiiModels/RadiometricTargetSearch.php]: Please change "radioemtric" for "radiometric". 
Please uppercase the first word of the sentence

File: models/yiiModels/YiiRadiometricTargetModel.php [hunk 2]
```diff
@@ -0,0 +1,493 @@
...
+
+        if (!$this->isNewRecord) {
+            $elementForWebService[YiiRadiometricTargetModel::URI] = $this->uri;
+        }
+
+        $elementForWebService[YiiRadiometricTargetModel::LABEL] = $this->label;
+
+        $properties = [];
+
+        $properties[] = [
```
Comment (annetireau) [inline models/yiiModels/YiiRadiometricTargetModel.php]: Could you use constant for  "relation", "value", etc.

File: models/yiiModels/YiiRadiometricTargetModel.php [hunk 3]
```diff
@@ -0,0 +1,493 @@
...
+            "relation" =>  Yii::$app->params['hasBrand'],
```
Comment (annetireau) [inline models/yiiModels/YiiRadiometricTargetModel.php]: Could you abstract all specific keys ? hasBrand, inServiceDate, etc.

File: views/radiometric-target/_form.php [hunk 4]
```diff
@@ -0,0 +1,169 @@
...
+                'value' => [],
+            ],
+            'pluginOptions' => [
+                'allowClear' => false
+            ],
+        ]);
+    ?>
+    
+    <script>
+        function rtChangeFigure(value) {
```
Comment (annetireau) [inline views/radiometric-target/_form.php]: you should change the name of this function, maybe remove rt

Comment (vincentmigot) [reply views/radiometric-target/_form.php]: I change it to radiometricTargetChangeFigure

**Pull request merged**